packages/generator/src/sdk-metadata/sdk-metadata-types.ts

```diff
@@ -17,6 +17,14 @@ export interface SdkMetadataHeader {
  */
 export interface Client {
   language: 'java' | 'javascript';
+  /**
+   * Status of the service.
+   * `certified` means we have a pregeneratedLibrary.
+   * `verified` means we have generated and transpiled the client
+   * `unknown` means we have no information.
+   * @memberof Client
+   */
+  serviceStatus: 'certified' | 'verified' | 'unknown';
```

Comment:
so unknown is the default status and might also applies to some services with known issues with generation.

Reply:
yes unknown will be the value for the generic response when we have no data. Potentially we add even a `error` status in case we have a problem. Currently the exception are not handled to make this possible but we will do this later.